.circleci/config.yml

```diff
@@ -16,6 +16,19 @@ workflows:
             tags:
               only: /^v.*/
 
+      - architect/push-to-docker:
+          context: "architect"
+          name: push-app-admission-controller-to-docker
+          image: "docker.io/giantswarm/app-admission-controller"
+          username_envar: "DOCKER_USERNAME"
+          password_envar: "DOCKER_PASSWORD"
+          requires:
+            - go-build
```

Comment:
Doesn't exist

Reply:
@MarcelMue My bad I did this too fast. I changed the name to `go-build` to align with other projects.

PTAL